RecoBTag/DeepFlavour/plugins/ChargedCandidateConverter.h

```diff
@@ -30,7 +30,19 @@ namespace btagbtvdeep {
                                             const reco::Jet & jet,
                                             const TrackInfoBuilder & track_info,
                                             const float & drminpfcandsv,
-                                            ChargedCandidateFeatures & c_pf_features) {
+                                            ChargedCandidateFeatures & c_pf_features,
+					    const bool flip) {
+
+	float TrackSip2dVal = track_info.getTrackSip2dVal();
+	float TrackSip2dSig = track_info.getTrackSip2dSig();
+	float TrackSip3dVal = track_info.getTrackSip3dVal();
+	float TrackSip3dSig = track_info.getTrackSip3dSig();
+	if(flip == true){
+	  TrackSip2dVal = -1.0*TrackSip2dVal;
```

Comment:
```- TrackSip2dVal;``` should be good enough

Reply:
Done in the new PR at https://github.com/cms-sw/cmssw/pull/23467/files#diff-898f4df86b3e380a58ff390ca3e02194R23